src/FileHelper.php

```diff
@@ -576,13 +576,13 @@ public static function lastModifiedTime(string|RecursiveDirectoryIterator ...$pa
         return $time;
     }
 
+    /**
+     * @see https://www.php.net/manual/function.filemtime.php
+     */
     private static function modifiedTime(string $path): ?int
     {
-        if (false !== $timestamp = filemtime($path)) {
-            return $timestamp;
-        }
-
-        return null;
+        $timestamp = @filemtime($path);
```

Comment:
It is better to be consistent with https://github.com/yiisoft/files/blob/master/src/FileHelper.php#L71 error handling.

Reply:
`lastModifiedTime()` and this method returns null when it was not possible to get modified time. So we shouldn't throw exception on fail, isn't we?

Reply:
Yes. I mean, we can either suppress the error with `@` or handle it as in the code I've linked. I prefer handling it.

Reply:
Set/unset handler needs to done for each of files, there may be many of them.

Reply:
OK. Let's leave it like that for now.